docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1d.md

```diff
@@ -0,0 +1,82 @@
+               - Check if the hvac system serves more than one zone, if it does then carry on, if not then go to Else: `if len(zone_list_hvac_sys_x) >1:`  
+                   - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["Terminal_Unit_List"]`  
+                   - create a variable for the total terminal air flow for the hvac system - this total terminal airflow is used to divide exhaust proportionally across zones: `total_terminal_air_flow = 0`
+                   - create a variable to hold the terminal airflow for this zone: `zone_primary_air_flow = 0`
+                   - For each terminal unit served by the HVAC system: `for terminal in terminal_list_hvac_sys_x:`  
+                       - Get the primary air flow: `primary_air_flow_terminal = terminal.primary_airflow`  
+                       - Add to the total cfm across all terminals served by the HVAC system: `total_terminal_air_flow = total_terminal_air_flow + primary_air_flow_terminal`  
+                       - Check if the terminal unit is associated with this zone: `if terminal in zone.terminals:`   
+                           - Add to the primary flow for the zone for this hvac system:  `zone_primary_air_flow = zone_primary_air_flow + primary_air_flow_terminal`  
+                   - Get the HVAC system's FanSystem: `fan_system = hvac.fan_system`
+                   - Apportion the exhaust airflow to the zone based on the terminal unit primary flow for the zone if hvac_system_total_exhaust_airflow > 0 and zone_primary_air_flow > 0: `if( zone_primary_air_flow > 0 && hvac_system_total_exhaust_airflow > 0 ): zone_total_exhaust += hvac_system_total_exhaust_airflow / zone_primary_air_flow`
+               - Else, the HVAC system serves a single zone: `Else:`  
+                   - Add the hvac system exhaust flow to zone_total_exhaust: `zone_total_exhaust += hvac_system_total_exhaust_airflow`  
+       - now check whether this zone meets the threshold of having greater than 15,000 cfm exhaust flow: `if zone_total_exhaust > 15000:`
+       - add the zone_total_exhaust to building_total_lab_exhaust: `building_total_lab_exhaust = building_total_lab_exhaust + zone_total_exhaust`
+     - now check whether the building wide total lab exhaust is greater than 15,000 cfm: `if building_total_lab_exhaust > 15000:`
+       - set the result to YES: `result = YES`
+
+
+
+**Returns** `result`
+
+**Notes**
+1.  This function determines whether the zone is a lab zone, and whether the building has total lab exhaust greater than 15,000 cfm.  We can determine with precision when there is less than 15,000cfm of lab exhaust, but not necessarily all air that is exhausted from a lab zone is classified as lab exhaust.  Therefore, when we do rule evaluation, we can only give a 100% positive identification of a lab zone when the lab zones have a total of more than 15,000 cfm of zone exhaust.
+
```

Comment:
I would also add a note that when a lab zone is served by HVAC system with exhaust fans serving both lab and non-lab zones, we assume that exhaust flow rate is allocated to the lab zones in proportion to the primary air flow delivered to these zones comared to the total primary air flow delivered to all zones by this system.

Reply:
updated